CONTRIBUTING.md

```diff
@@ -47,24 +51,15 @@ As of Go 1.9 this is simplified to `go test ./...`.
 
 ## Manual Testing against Exercism
 
-You can build whatever is in your local, working copy of the CLI without overwriting your existing Exercism
-CLI installation by using the `go build` command:
+If you want to test your changes while doing your everyday exorcism work you could do:
 
-```
-go build -o testercism exercism/main.go
-```
-
-This assumes that you are standing at the root of the exercism/cli repository checked out locally, and it will put a binary named `testercism` in your current working directory.
-
-You can call it whatever you like, but `exercism` would conflict with the directory that is already there.
-
-Then you call it with `./testercism`.
+On Unices:
 
-You can always put this in your path if you want to run it from elsewhere on your system.
+- `cd $GOPATH/src/github.com/exercism/cli/exercism && go build -o exercism main.go && mv exercism ~/bin`
 
-We highly recommend spinning up a local copy of Exercism to test against so that you can mess with the database (and so you don't accidentally break stuff for yourself in production).
+On Windows:
 
-[TODO: link to the nextercism repo installation instructions, and explain how to reconfigure the CLI]
+- ?? TODO
```

Comment:
@FabienTregan do you know what the windows command is to build?

Reply:
I usually build from a git bash :
```
$ cd $GOPATH/src/github.com/exercism/cli/
go build -o e.exe exercism/main.go
```
I tested this from a command shell and it seems to work :
```
cd /d %GOPATH%\src\github.com\exercism\cli
go build -o e.exe exercism\main.go
```

An since you were not speaking about the CLI but the server, I curently am not able to build the server on windows. I tracked what I managed to do and where I'm stuck here : https://github.com/exercism/v2-feedback/issues/209 (maybe the API is working now, I did not test)

Reply:
👍